lib/internal/streams/transform.js

```diff
@@ -244,3 +246,25 @@ Transform.prototype._read = function() {
     callback();
   }
 };
+
+Transform.by = function by(asyncGeneratorFn, options) {
+  let { promise, resolve } = createDeferredPromise();
```

Comment:
Nit: Can this be declared as `const`?

Reply:
I don't think so because these variables are reassigned values afterwards:
https://github.com/nodejs/node/blob/6ccafaba0e4e92ba7ed3582711bc29a9042c2030/lib/internal/streams/transform.js#L258